src/v/http/client.h

```diff
@@ -285,3 +285,29 @@ struct fmt::formatter<http::client::request_header> {
         return fmt::format_to(ctx.out(), "{}", s.str());
```

Comment:
```
commit 28682c8f58b6729bc9583c96f005b603e68a3afd
Author: Andrea Barbadoro <andrea@redpanda.com>
Date:   Fri Dec 23 10:45:43 2022 +0100

    http/client: added formatter for http::client::response_header. support 'l' modifier to print fields on one line
```

can you fix the commit subject length? suggested lengths are subject=60ish body=72ish. mostly just try to not have github wrap which i think happens at 70 characters

Reply:
I can do it, worst case I'll do it in 2023